static/js/page_params.js

```diff
@@ -0,0 +1 @@
+window.page_params = $(document.body).data("params");
```

Comment:
Not sure what happened to my comment on this, but we might want to remove that `.data` element after extracting the data from it.

Reply:
I’m unclear on whether that would actually save memory, but I suppose it doesn’t hurt to try.